docs/standard/microservices-architecture/architect-microservice-container-applications/asynchronous-message-based-communication.md

```diff
@@ -3,7 +3,7 @@ title: Asynchronous message-based communication
 description: .NET Microservices Architecture for Containerized .NET Applications | Asynchronous message-based communication
```

Comment:
👍 I agree. This can be done while updating/submitting each PR. But we cannot review/change ALL the descriptions for the eBook now. We need to move fast on the update.
Let's do this "on-the-go", only for every page being updated, ok?

Reply:
Ok. All sections have had some change so far, though.